i6_models/parts/frontend/README.md

```diff
@@ -0,0 +1,5 @@
+# Different front-ends for acoustic encoders
+
+- Two classes are required
+- `Config` class inherits from `ModelConfiguration`
+- Actual Model inherits from `nn.Module` from `torch`
```

Comment:
I don't exactly understand what this readme is telling me.

Two classes for what? For the user? Which two classes?

Why is the comment for `Config` relevant?

Why is the comment for `nn.Module` relevant?

Reply:
for now this is just stubs. I ll extend